AndroidApps/android-core/src/main/java/net/discdd/crashreports/LocalReportSender.java

```diff
@@ -31,17 +31,20 @@ public LocalReportSender(CoreConfiguration coreConfiguration) {
         config = coreConfiguration;
     }
 
-    @Override
+    @Override //might crash reports not be working on transports at all???
     public void send(Context context, CrashReportData errorContent) throws ReportSenderException {
         Path rootDir = context.getApplicationContext().getDataDir().toPath();
         logger.log(INFO, "root directory for acra class " + rootDir);
         Path clientDestDir = rootDir.resolve("to-be-bundled");
         if (clientDestDir.toFile().exists()) {
             logger.log(INFO, "We are writing crash report to client device internal storage");
             rootDir = clientDestDir;
-        } else {
+        } else if (context.getApplicationContext().getExternalFilesDir(null).exists()) {
```

Comment:
perhaps we should change this to always write crashreports to the internal director: .getFilesDir(). that would simplify this all and always work.